graphql_api/types/test_results_aggregates/test_results_aggregates.py

```diff
@@ -0,0 +1,23 @@
+from ariadne import ObjectType
+
+test_results_aggregates_bindable = ObjectType("TestResultsAggregates")
+
+
+@test_results_aggregates_bindable.field("totalRunTime")
+def resolve_name(obj, _) -> float:
```

Comment:
nit: I think our naming convention for these functions is resolve_bindable_field_name